tools/IncludeBin/IncludeBin.cpp

```diff
@@ -0,0 +1,33 @@
+#include <cstdio>
```

Comment:
just throwing it out there :D For utility things like that we might do python script as well.